docs/content/logging-and-monitoring/service-insight.md

```diff
@@ -39,9 +39,9 @@ If you're using *Helm* to install the Ingress Controller, to enable Service Insi
 
 The Service Insight provides the following statistics:
 
-* Total number of VS
-* Number of VS in 'Down' state
-* Number of VS in 'Healthy' state
+* Total number of VS and TS
+* Number of VS and TS in 'Down' state
+* Number of VS and TS in 'Healthy' state
```

Comment:
It'll be ideal to keep this in line with:
`{"Total": 1, "Up": 1, "Unhealthy": 0}`

Reply:
applied